vorestation.dme

```diff
@@ -2401,6 +2401,11 @@
 #include "code\modules\vore\fluffstuff\custom_items_vr.dm"
 #include "code\modules\vore\fluffstuff\custom_mecha_vr.dm"
 #include "code\modules\vore\fluffstuff\custom_permits_vr.dm"
+#include "code\modules\vore\fluffstuff\guns\bsharpoon.dm"
+#include "code\modules\vore\fluffstuff\guns\dominator.dm"
+#include "code\modules\vore\fluffstuff\guns\protector.dm"
+#include "code\modules\vore\fluffstuff\guns\pummeler.dm"
+#include "code\modules\vore\fluffstuff\guns\sickshot.dm"

```

Comment:
I agree.  Deese separate files are nice.
This code looks like it will work to me.